api/source/service/CollectionService.js

```diff
@@ -756,7 +756,7 @@ exports.getChecklistByCollectionStig = async function (collectionId, benchmarkId
     // Non-current revision
     if (revisionStr !== 'latest') {
       joins.splice(2, 1, 'left join revision rev on sa.benchmarkId=rev.benchmarkId')
-      const results = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
+      const results = dbUtils.extractRevisionDetails(revisionStr)
       predicates.statements.push('rev.version = :version')
       predicates.statements.push('rev.release = :release')
       predicates.binds.version = results[1]
```

Comment:
There are several instances where this structure is used too, and these will need to change to the 0,1 items in the array as well.